Move Service Addressing from appendix to dedicated section (#150)
scionassociation/scion-cp_I-D: draft-dekater-scion-controlplane.md

In case of failure, RPC calls return an error as specified by the RPC framework. That is, a non-zero status code and an explanatory string.

## Control Service Discovery

> The Control Plane RPC APIs rely on QUIC connections over UDP/SCION (see {{I-D.dekater-scion-dataplane}} which requires a connection to be initiated to identify the relevant peer (service resolution) and to select a path to it. Since the Control Service is itself the source of path segment information, the following bootstrapping processes apply:

Reviewer: > ... requires a connection to be initiated to identify the relevant peer .... 

This sentence is not really clear and IMHO too long. 

```suggestion
The Control Plane RPC APIs rely on QUIC connections over UDP/SCION (see {{I-D.dekater-scion-dataplane}}. Establishing such connection requires the initiator to identify the relevant peer (service resolution) and to select a path to it. Since the Control Service is itself the source of path segment information, the following bootstrapping processes apply:
```

The SCION Control Plane RPC APIs rely on QUIC connections carried by the SCION dataplane. The main difference between QUIC over native UDP and QUIC over UDP/SCION is the need for a UDP/SCION connection initiator to identify the relevant peer (service resolution) and to select a path to it. Since the Control Service is itself the source of path segment information, the following bootstrapping strategies apply:
